### ISSUE TITLE
'for' helper should run if 'from' and 'to' are equal

### ISSUE THREAD

**Issue by flyingL123** [bigcommerce/paper-handlebars#28]
I've reported this before, but just came across it again, so here it goes:

https://github.com/bigcommerce/paper/issues/90

It is currently impossible to run the `for` helper one time. It's coded not to run if `from <= to`. However, the `for` loop in the helper runs from `i = from` to `i <= to`. Either the for loop needs to change, or the check on line 28 should be changed to `if (to < from) {`:

https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/for.js#L28

In other words, if `from = 1` and `to = 2`, the helper runs twice. But if `from = 1` and `to = 1`, the helper doesn't run at all. It needs to be updated so it can run once if necessary.
